src/Symfony/Component/HttpFoundation/Tests/Session/Storage/Handler/Fixtures/storage.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+require __DIR__.'/common.inc';
+
+use Symfony\Component\HttpFoundation\Session\Flash\FlashBag;
+use Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage;
+
+$storage = new NativeSessionStorage();
+$storage->setSaveHandler(new TestSessionHandler());
+$flash = new FlashBag();
+$storage->registerBag($flash);
```

Comment:
Here you test NativeSessionStorage with one flash bag. That is good.

But I miss a test where you have the full symfony framework, where the standard bags are automatically added, including a metaData bag.

Will this test catch it if the metaData bag is not removed when all the other bags are empty?

Reply:
@TerjeBr the metadata bag is automatically added by the constructor of the nativesessionstorage. there is no difference to full symfony. see nativesessionstorage::save that deletes empty bags.